.gitignore

```diff
@@ -45,9 +45,10 @@ cmd/kas-fleet-manager/kas-fleet-manager
 # Ignore secret files
 /secrets/db.ca_cert
 /secrets/aws.*
-/secrets/vault.*
```

Comment:
Is this correct? I think vault was something that was added by connectors

Reply:
See:

```

msoriano@localhost:~/go/src/github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager (...)$ git log secrets/vault/.gitignore
commit 6ff911ee7a5048045af6bd7749dee8bb7bf46ad8
Author: Dhiraj Bokde <dhirajsb@gmail.com>
Date:   Tue Nov 1 00:07:13 2022 +0000

    feat(connectors): related to APPSRE-4651, moved vault credentials config files
``

Reply:
No, clearly not. I think a rename went wrong and this should stay as well.